JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/converter/README.md

```diff
@@ -0,0 +1,203 @@
+
+## Filled out CSV file-based template
+Suppose you followed the instructions for filling out the template in the TBox Metadata, Classes, Object Properties and Data Properties sections. In that case, your CSV file will have precisely the following information.
+
+|Source,Type,Target,Relation,Domain,Range,Quantifier,Comment,Defined By,Label|
+|----------------------------------------------------------------------------|
+|OntoKin,TBox,http://www.theworldavatar.com/kg/ontokin, https://www.w3.org/2007/05/powder-s#hasIRI, , , , , , |
+|OntoKin,TBox,1,http://www.w3.org/2002/07/owl#versionInfo, , , , , , |
+|OntoKin,TBox,OntoKin is an ontology developed for representing chemical kinetic reaction mechanisms,http://www.w3.org/2000/01/rdf-schema#comment, , , , , , |
+|OntoKin,TBox,http://theworldavatar.com/ontology/ontocape/OntoCAPE.owl,http://www.w3.org/2002/07/owl#imports, , , , , , |
+|ReactionMechanism,Class, , , , , ,A reaction mechanism refers to a set of elementary reactions., http://www.theworldavatar.com/ontology/ontokin/OntoKin.owl, Reaction Mechanism|
+|Phase,Class, , , , , ,A phase of a substance is a form of matter., http://www.theworldavatar.com/ontology/ontokin/OntoKin.owl, Phase|
+|GasPhase,Class,Phase,IS-A, , , ,A continuous gaseous phase.,http://www.theworldavatar.com/ontology/ontokin/OntoKin.owl, Gas Phase|
+|SitePhase,Class,Phase,IS-A, , , ,A phase that exists at the interface between the gas phase and a bulk phase.,http://www.theworldavatar.com/ontology/ontokin/OntoKin.owl, Site Phase|
+|BulkPhase,Class,Phase,IS-A, , , ,A solid phase that is contiguous with site phases and remote from the gas phase., http://www.theworldavatar.com/ontology/ontokin/OntoKin.owl,Bulk Phase|
+|Species,Class, , , , , ,An ensemble of chemically identical molecular entities (McNaught & Wilkinson - 1997).,http://www.theworldavatar.com/ontology/ontokin/OntoKin.owl,Species|
+|Element,Class, , , , , ,An atom or isotope.,http://www.theworldavatar.com/ontology/ontokin/OntoKin.owl,Element|
+|Material,Class, , , , , ,A substance that contains at least one site or bulk phase.,http://www.theworldavatar.com/ontology/ontokin/OntoKin.owl,Material|
+|RateCoefficient,Class,http://www.theworldavatar.com/ontology/ontocape/chemical_process_system/CPS_behavior/behavior.owl#ReactionRateCoefficient,EQUIVALENT-TO, , , ,The coefficients used to evaluate the reaction rate expression.,http://www.theworldavatar.com/ontology/ontokin/OntoKin.owl,Rate Coefficient |
+|existsIn,Object Property, , ,SitePhase UNION BulkPhase,Material,only,A relation between a site phase or bulk phase and a material in which they exist.,http://www.theworldavatar.com/ontology/ontokin/OntoKin.owl, exists in|
+|containedIn,Object Property, , ,GasPhase UNION Material,ReactionMechanism,exactly 1,A relation that identifies that a gas phase or material is contained in a reaction mechanism.,http://www.theworldavatar.com/ontology/ontokin/OntoKin.owl,contained in|
```

Comment:
Is that true by now?
I remember that previously Object Properties required 2 row entries in case a cardinality restricyion should be created. Is that solved now?

Reply:
Yes, it is. Now the code can represent the domain range and cardinality restriction from a single line.